gramjs/events/NewMessage.js

```diff
@@ -97,10 +97,8 @@ class NewMessage extends EventBuilder {
 
         // Check if the message is incoming or outgoing, and if
         // we want to accept whichever one it is
-        if (message.out) {
-            if (!this.outgoing) return false
-        } else {
-            if (!this.incoming) return false
+        if (message.out && (!message.outgoing || !message.incoming)) {
```

Comment:
you probably wanted to use `this.outgoing` instead of `message.outgoing`

Reply:
Reverted this back to the old code